src/boundaries/edgetracking.jl

```diff
@@ -1,11 +1,31 @@
-export edgetracking, bisect_to_edge
+export edgetracking, bisect_to_edge, EdgeTrackingResults
 
 """
-    edgetracking(ds::DynamicalSystem, attractors::Dict; kwargs...) -> edge, track1, track2
+    EdgeTrackingResults(edge, track1, track2, time, bisect_idx)
+Data type that stores output of the [`edgetracking`](@ref) algorithm.
+
+## Fields
+* `edge::StateSpaceSet`: the pseudo-trajectory representing the edge track
+* `track1::StateSpaceSet`: the parallel pseudo-trajectory tracking the edge within basin 1
+* `track2::StateSpaceSet`: the parallel pseudo-trajectory tracking the edge within basin 2
+* `time::Vector`: time points of the above `StateSpaceSet`s
+* `bisect_idx::Vector`: indices of `time` at which a re-bisection was done
+"""
+struct EdgeTrackingResults
+    edge::StateSpaceSet
```

Comment:
`StateSpaceSet{D, T}` instead. `StateSpaceSet` is an abstract type. Parameterize `EdgeTrackingResults{D,T}`

Reply:
Ok!